etc/kayobe/ansible/build-ofed.yml

```diff
@@ -0,0 +1,103 @@
+---
+- name: Build OFED packages
+  become: true
+  hosts: ofed-builder
+  gather_facts: false
+  tasks:
+    - name: Extend the home logical volume
+      community.general.lvol:
+        vg: rootvg
+        lv: lv_home
+        size: +2G
+        resizefs: true
+
+    - name: Extend the temporary logical volume
+      community.general.lvol:
+        vg: rootvg
+        lv: lv_var_tmp
+        size: +2G
+        resizefs: true
+
+    - name: Install package dependencies
+      ansible.builtin.dnf:
+        name:
+          - kpartx
+          - perl
+          - rpm-build
+          - automake
+          - patch
+          - kernel
+          - kernel-devel
+          - autoconf
+          - pciutils
+          - kernel-modules-extra
+          - kernel-rpm-macros
+          - lsof
+          - libtool
+          - tk
+          - gcc-gfortran
+          - tcl
+          - createrepo
+          - cmake-filesystem
+          - libnl3-devel
+          - python3-devel
+        state: latest
+        update_cache: true
+
+    - name: Set GRUB default index to saved
+      ansible.builtin.lineinfile:
+        path: /etc/default/grub
+        line: "GRUB_DEFAULT=saved"
+
+    - name: Set update default index in kernel sysconfig
+      ansible.builtin.lineinfile:
+        path: /etc/sysconfig/kernel
+        line: "UPDATEDEFAULT=yes"
+        create: true
+
+    - name: Update the default kernel entry
+      ansible.builtin.shell:
+        cmd: |
+          grubby --set-default /boot/$(rpm -qa kernel-devel | sed 's/kernel-devel/vmlinuz/g')
+          grub2-mkconfig -o /boot/grub2/grub.cfg
+
+    - name: Disable noexec in temporary file system
+      ansible.builtin.replace:
+        path: /etc/fstab
+        regexp: 'noexec,'
+
+    - name: Reboot builder to apply kernel update
+      ansible.builtin.reboot:
+        reboot_timeout: 600
+
+    - name: Add DOCA host repository
```

Comment:
The name of this task doesn't appear to match the content. Is it installing a package that includes the repo?

Reply:
Yes, that package installs the doca-host repository which is used for the doca-ofed installation process. The name could be clearer though.

Reply:
Changed to `Add DOCA host repository package` in a941098b.